.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Bug Report
+about: Create a report to help us solve bugs!
+labels: bug
+---
+
+<!--
+        ✰  Thanks for opening an issue! ✰
+  Before submitting please review the template.
+  Please also ensure that this is not a duplicate issue :)
+-->
+
+## Summary
+
+<!-- Concisely describe the issue -->
+
+## Steps to Reproduce
+
+<!-- What commands in order should someone run to reproduce your problem? -->
+
```

Comment:
After the Steps to Reproduce, I suggest calling out "Actual Observed Behavior" stating:
<!-- A clear and concise description of the specific misbehavior -->

In practices, users may report in the summary things like x is blocking y or, x is not working. Thus, having a dedicated space to specifically, identify the actual observed behavior becomes important.

Reply:
Yes, this was missing indeed!